### ISSUE TITLE
Project dead ?

### ISSUE THREAD

**Issue by boustanihani** [rethinkdb/horizon#883]
Is this project dead?

The last commit was on Nov 3, 2016.

**Comment by threefoldo** [comment]
This implementation is quite limited, only a few simple queries are supported. Maybe they are working on something more general.

**Comment by nosykretts** [comment]
#850

**Comment by emeagenciadigital** [comment]
any news from the team?

**Comment by Extarys** [comment]
They shut down but Linux Foundation took over the RethinkDB company
I really hope the project will continue <3

**Comment by prateekvarma** [comment]
:worried:

**Comment by ItsEcholot** [comment]
For anyone trying to access the official project website http://horizon.io (without https) works. 

Can anyone from the core contribution / dev team take a stand on the projects future?

**Comment by sp90** [comment]
So sad to see this die out.....

**Comment by cevarief** [comment]
This is cool. Someone does not maintain this any longer?

**Comment by justsml** [comment]
So sad... such good tech. 😿

**Comment by shigurenimo** [comment]
I hope development will resume.

**Comment by Extarys** [comment]
@ItsEcholot "Can anyone from the core contribution / dev team take a stand on the projects future?"

@deontologician @neumino @Tryneus any idea who we can tag/ask? :D

**Comment by neumino** [comment]
The project is basically community driven now (as any other RethinkDB projects)

cc @marshall007 @KittyBot

**Comment by Extarys** [comment]
Thanks for dropping by @neumino, I knew you would know who to mention in this to get people their answers.

Yes it is quite sad :(

**Comment by LennonSantos** [comment]
quite sad :(

**Comment by nurmuhammet-ali** [comment]
The official site is no longer works seems like..

**Comment by deontologician** [comment]
horizon.io is still up

**Comment by srh** [comment]
Huh, I get 
<img width="932" alt="screen shot 2017-12-04 at 6 36 14 pm" src="https://user-images.githubusercontent.com/258437/33587256-10a0925a-d922-11e7-9914-69a7ef6629d9.png">

**Comment by deontologician** [comment]
Oh, you're right. I had to bust my cache. Whoops!

**Comment by mmerdes** [comment]
Any chance to turn the documentation site on again or move it elsewhere?

**Comment by jeremy-wischusen** [comment]
Same question here. Do the docs exist anywhere else?

**Comment by deontologician** [comment]
I don't know, technically the source of the webpage should be a part of one of the repos for rethinkdb

**Comment by mmerdes** [comment]
as far as i can tell the sources for the docs are here:
https://github.com/rethinkdb/horizon-docs

**Comment by srh** [comment]
I'd like to host the docs somewhere, but I have no idea how you're supposed to turn that repo into a website.  Its README doesn't really help.

**Comment by srh** [comment]
Alright, I figured out how to turn that repo into a website.  I went through and edited all the links to point at .html files and then I made a little static website demonstration script.

I've got the docs hosted at http://samuelhughes.com/rethinkdb/horizon-docs/docs/

If you pester me I might improve the CSS a bit :-)

The source for building the static website is at my fork https://github.com/srh/horizon-docs/

**Comment by michalvankodev** [comment]
I still hope this project is not dead. At least there could be a statement from someone from rethinkdb org. about this.

**Comment by srh** [comment]
(Not from rethinkdb org.)  You can evaluate for yourself whether the project is dead or not based on activity.  If you want it to not be dead, you have the power to make that happen.

**Comment by michalvankodev** [comment]
Yes.. Where can I pay for the hosting of the docs ?.. Docs being available would be the best for this project

**Comment by srh** [comment]
You know what, maybe it's dead.

**Comment by srh** [comment]
I don't even think it's about hosting cost, somebody somewhere needs to push a button.

**Comment by srh** [comment]
I've pinged people who might know something about the website in a way they're more likely to notice.

**Comment by Extarys** [comment]
Thanks. At least the doc is up.

**Comment by Nymoth** [comment]
Found the original horizon.io page saved on web archive. A bit slow tho.
Hope it helps.

https://web.archive.org/web/20170707182557/http://horizon.io:80/

**Comment by beenotung** [comment]
@Nymoth if we can update the website link of the github repo to this web archive, it would be helpful for many others.

**Comment by michalvankodev** [comment]
I've just ran into the #533 I've noticed that there are plenty of fixes done (half a year of development) on `next` branch. Is anyone using the latest build?

**Comment by mhlr** [comment]
RethinkDB has found a new home at the Linux Foundation.
https://rethinkdb.com/blog/rethinkdb-joins-linux-foundation/
Maybe Horizon will follow. 

Also looks like a few forks have been active fairly recently
https://github.com/rethinkdb/horizon/network

**Comment by elyngved** [comment]
I guess Horizon has been... sunset? 🤡

**Comment by andrwwbstr** [comment]
We have partially restored the horizon site from the web archive. Some content is missing and there are probably some broken links.
https://support.arcx.com/horizon/
http is also supported at the moment.

**Comment by beenotung** [comment]
I refactor the website from web archive and host it on a CDN.
https://horizon-io.surge.sh 

the repo at 
https://github.com/rethinkdb-mirror/horizon.io

**Comment by danieagle** [comment]
Note: Old RethinkDB changed to RethinkDB-legacy 
and a new healthy developing taking place in RethinkDB as a Rebrand of RebirthDB.
  With This, I Hope many projects that depend on RethinkDB will revamp without fear. :-)
